packages/module/src/ConsoleShared/src/components/spotlight/spotlight.scss

```diff
@@ -36,14 +36,14 @@
     mix-blend-mode: hard-light;
   }
   &__element-highlight-noanimate {
-    border: var(--pf-v5-global--BorderWidth--xl) solid var(--pf-v5-global--palette--blue-200);
-    background-color: var(--pf-v5-global--palette--black-500);
+    border: var(--pf-v6-global--BorderWidth--xl) solid var(--pf-v6-global--palette--blue-200);
+    background-color: var(--pf-v6-global--palette--black-500);
     z-index: 9999;
   }
   &__element-highlight-animate {
     pointer-events: none;
     position: absolute;
-    box-shadow: inset 0px 0px 0px 4px var(--pf-v5-global--palette--blue-200);
+    box-shadow: inset 0px 0px 0px 4px var(--pf-v6-global--palette--blue-200);
```

Comment:
We don't have a blue shadow color, I wonder if the generic brand color token would work - `--pf-t--global--color--brand--default`